exchangelib/util.py

```diff
@@ -291,7 +301,7 @@ def post_ratelimited(protocol, session, url, headers, data, timeout=None, verify
             try:
                 r = session.post(url=url, headers=headers, data=data, allow_redirects=False, timeout=timeout,
                                  verify=verify)
-            except (requests.exceptions.ChunkedEncodingError, requests.exceptions.ConnectionError, ConnectionResetError,
+            except (requests.exceptions.ChunkedEncodingError, requests.exceptions.ConnectionError,
```

Comment:
This needs to be a conditional yes, it was a relic from getting the whole thing running for a start but that was always intended to be included in the end but somehow was left out. Good catch!